README.md

```diff
@@ -1 +1,64 @@
-# first-capstone-summit
\ No newline at end of file
+# html-css-js-capstone-project
+
+This is my final project for HTML & CSS MODULE from the microverse curriculum
+
```

Comment:
- [ ] Following the project requirements, you should record a video presenting the features of the project. Please make sure that:
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.